lib/recurly/base.php

```diff
@@ -304,7 +304,8 @@ public function getLinks() {
     'transaction_error' => 'Recurly_TransactionError',
     'unit_amount_in_cents' => 'Recurly_CurrencyList',
     'usage' => 'Recurly_Usage',
-    'usages' => 'Recurly_UsageList'
+    'usages' => 'Recurly_UsageList',
+    'verify' => 'Recurly_Verify'
```

Comment:
What is this for? I don't see a `Recurly_Verify` class in the PR.

Reply:
my b. it's a vestige. removing it now.